### PR TITLE
AWS: add scylla_io_setup preset parameters for ARM instances

### LINKED ISSUE #9493: scylla can't start when running in ARM based ami image
Instance type:  c6gd.2xlarge
ami-id: ami-0f883f1042d6d7df9
instance-id: i-0f386c8ae92413d3e

```
scyllaadm@ip-172-30-0-70:~$ sudo systemctl status scylla-image-setup.service 
● scylla-image-setup.service - Scylla Cloud Image Setup service
     Loaded: loaded (/lib/systemd/system/scylla-image-setup.service; enabled; vendor preset: enabled)
     Active: active (exited) since Mon 2021-10-18 09:13:09 UTC; 4min 27s ago
    Process: 613 ExecStart=/opt/scylladb/scylla-machine-image/scylla_image_setup (code=exited, status=0/SUCCESS)
   Main PID: 613 (code=exited, status=0/SUCCESS)

Oct 18 09:12:20 ip-172-30-0-70 scylla_image_setup[7300]:                 #9  0x0000aaaae302a644 _start (bash + 0x32644)
Oct 18 09:12:20 ip-172-30-0-70 scylla_image_setup[7300]:                 #10 0x0000aaaae302a644 _start (bash + 0x32644)
Oct 18 09:12:20 ip-172-30-0-70 scylla_image_setup[7300]: systemd-coredump is working finely.
Oct 18 09:12:48 ip-172-30-0-70 scylla_image_setup[7458]: 5234+0 records in
Oct 18 09:12:48 ip-172-30-0-70 scylla_image_setup[7458]: 5234+0 records out
Oct 18 09:12:48 ip-172-30-0-70 scylla_image_setup[7458]: 5488246784 bytes (5.5 GB, 5.1 GiB) copied, 27.2287 s, 202 MB/s
Oct 18 09:13:09 ip-172-30-0-70 scylla_image_setup[7479]: Setting up swapspace version 1, size = 5.1 GiB (5488242688 bytes)
Oct 18 09:13:09 ip-172-30-0-70 scylla_image_setup[7479]: no label, UUID=d09fb139-3f06-4ca4-b5e8-6dcbb6d7e7d7
Oct 18 09:13:09 ip-172-30-0-70 scylla_image_setup[7516]: Created symlink /etc/systemd/system/multi-user.target.wants/swapfile.swap → /etc/systemd/system/swapfile.swap.
Oct 18 09:13:09 ip-172-30-0-70 systemd[1]: Finished Scylla Cloud Image Setup service.
```

Although Scylla service is not running with the following error
```
scyllaadm@ip-172-30-0-70:~$ sudo systemctl status scylla-server.service 
● scylla-server.service - Scylla Server
     Loaded: loaded (/lib/systemd/system/scylla-server.service; enabled; vendor preset: enabled)
    Drop-In: /etc/systemd/system/scylla-server.service.d
             └─capabilities.conf, dependencies.conf, mounts.conf, sysconfdir.conf
     Active: failed (Result: exit-code) since Mon 2021-10-18 09:14:06 UTC; 4min 16s ago
    Process: 7549 ExecStartPre=/opt/scylladb/scripts/scylla_prepare (code=exited, status=0/SUCCESS)
    Process: 7645 ExecStart=/usr/bin/scylla $SCYLLA_ARGS $SEASTAR_IO $DEV_MODE $CPUSET $MEM_CONF (code=exited, status=1/FAILURE)
    Process: 7674 ExecStopPost=/opt/scylladb/scripts/scylla_stop (code=exited, status=0/SUCCESS)
   Main PID: 7645 (code=exited, status=1/FAILURE)

Oct 18 09:13:10 ip-172-30-0-70 scylla_prepare[7615]: Writing '4096' to /proc/sys/net/core/somaxconn
Oct 18 09:13:10 ip-172-30-0-70 scylla_prepare[7615]: Writing '4096' to /proc/sys/net/ipv4/tcp_max_syn_backlog
Oct 18 09:13:10 ip-172-30-0-70 scylla_prepare[7615]: Clocksource setting not available or not needed for this architecture. Not tuning
Oct 18 09:13:10 ip-172-30-0-70 scylla[7645]: Scylla version 4.6.dev-0.20211007.e88891a8a with build-id 360f4a97cda8a3d865af40042903f42373c9912d starting ...
Oct 18 09:13:10 ip-172-30-0-70 scylla[7645]: command used: "/usr/bin/scylla --log-to-syslog 1 --log-to-stdout 0 --default-log-level info --network-stack posix --io-properties-file=/etc/scylla.d/io_properties.yaml --cpuset 0-7 --lock-memory=1"
Oct 18 09:13:10 ip-172-30-0-70 scylla[7645]: parsed command line options: [log-to-syslog, (positional) 1, log-to-stdout, (positional) 0, default-log-level, (positional) info, network-stack, (positional) posix, io-properties-file: /etc/scylla.d/io_properties.yaml, cpuset, (position>
Oct 18 09:13:10 ip-172-30-0-70 scylla[7645]: Could not initialize seastar: YAML::InvalidNode (invalid node; first invalid key: "read_bandwidth")
Oct 18 09:14:06 ip-172-30-0-70 systemd[1]: scylla-server.service: Main process exited, code=exited, status=1/FAILURE
Oct 18 09:14:06 ip-172-30-0-70 systemd[1]: scylla-server.service: Failed with result 'exit-code'.
Oct 18 09:14:06 ip-172-30-0-70 systemd[1]: Failed to start Scylla Server.
```

Part of  `/opt/scylladb/scylla-machine-image/libexec/scylla_image_setup` is to run `cloud_instance.io.setup()` which is located under `/opt/scylladb/scripts/scylla_util.py`
The actually command is `return run('/opt/scylladb/scripts/scylla_io_setup --ami', shell=True, check=True)`

When running this command manually nothing happend
```
scyllaadm@ip-172-30-0-70:~$ sudo /opt/scylladb/scripts/scylla_io_setup --ami
scyllaadm@ip-172-30-0-70:~$ 
```

When running:
```
scyllaadm@ip-172-30-0-70:~$ sudo /opt/scylladb/scripts/scylla_io_setup
tuning /sys/devices/pci0000:00/0000:00:1f.0/nvme/nvme1/nvme1n1
already tuned: /sys/devices/pci0000:00/0000:00:1f.0/nvme/nvme1/nvme1n1/queue/nomerges
tuning /sys/devices/pci0000:00/0000:00:1f.0/nvme/nvme1/nvme1n1
tuning /sys/devices/pci0000:00/0000:00:1f.0/nvme/nvme1/nvme1n1
tuning /sys/devices/pci0000:00/0000:00:1f.0/nvme/nvme1/nvme1n1
tuning /sys/devices/pci0000:00/0000:00:1f.0/nvme/nvme1/nvme1n1
INFO  2021-10-18 09:24:08,252 [shard 0] iotune - /var/lib/scylla/saved_caches passed sanity checks
INFO  2021-10-18 09:24:08,252 [shard 0] iotune - Disk parameters: max_iodepth=127 disks_per_array=1 minimum_io_size=512
Starting Evaluation. This may take a while...
Measuring sequential write bandwidth: 253 MB/s (deviation 11%)
Measuring sequential read bandwidth: 613 MB/s (deviation 43%)
Measuring random write IOPS: 49031 IOPS (deviation 27%)
Measuring random read IOPS: 119287 IOPS (deviation 33%)
Writing result to /etc/scylla.d/io_properties.yaml
Writing result to /etc/scylla.d/io.conf
scyllaadm@ip-172-30-0-70:~$ sudo systemctl restart scylla-server.service 
scyllaadm@ip-172-30-0-70:~$ nodetool status
Datacenter: us-east
===================
Status=Up/Down
|/ State=Normal/Leaving/Joining/Moving
--  Address      Load       Tokens       Owns    Host ID                               Rack
UN  172.30.0.70  278.24 KB  256          ?       6e440eb8-eef8-4439-a34f-ed034d734769  1b

Note: Non-system keyspaces don't have the same replication settings, effective ownership information is meaningless
scyllaadm@ip-172-30-0-70:~$ 
```
It's working

### PR DESCRIPTION
**Pull request by syuu1228**
Currently, scylla-server fails to start on ARM instances because scylla_io_setup does not have preset parameters even instance type added to 'supported instance'.
To fix this, we need to add io parameter preset on scylla_io_setup.

Also, we mistakenly added EBS only instances at a004b1da305a8a83d94e5ea585b6f41a69647640, need to remove them.
Instrances does not have ephemeral disk should be 'unsupported instance', we still run our AMI on it, but we print warning message on login prompt, and user requires to run scylla_io_setup.

Fixes #9493

### CODE REVIEW

File: dist/common/scripts/scylla_io_setup [hunk 1]
```diff
@@ -232,6 +232,47 @@ if __name__ == "__main__":
...
+                if idata.instance_size() == "medium":
+                    disk_properties["read_iops"] = 14808
+                    disk_properties["read_bandwidth"] = 77869147
+                    disk_properties["write_iops"] = 5972
+                    disk_properties["write_bandwidth"] = 32820302
+                elif idata.instance_size() == "large":
+                    disk_properties["read_iops"] = 29690
+                    disk_properties["read_bandwidth"] = 157712240
+                    disk_properties["write_iops"] = 12148
+                    disk_properties["write_bandwidth"] = 65978069
```
Comment (laurovenancio) [inline dist/common/scripts/scylla_io_setup:245]: The xlarge instance size is missing.

Comment (syuu1228) [reply dist/common/scripts/scylla_io_setup:245]: Oh, you are right, I will fix.

File: dist/common/scripts/scylla_io_setup [hunk 2]
```diff
@@ -232,6 +232,47 @@ if __name__ == "__main__":
                 disk_properties["read_bandwidth"] = 507338935 * nr_disks
                 disk_properties["write_iops"] = 57100 * nr_disks
                 disk_properties["write_bandwidth"] = 483141731 * nr_disks
+            elif idata.instance_class() in ("c6gd", "m6gd", "r6gd", "x2gd"):
+                if idata.instance_size() == "medium":
+                    disk_properties["read_iops"] = 14808
```
Comment (avikivity) [inline dist/common/scripts/scylla_io_setup:237]: We could simplify the code by creating a large dict of dict of namedtuple and then select from the dict:

```python
known_instance_io = {
    'i3': {
         'large': iorates(read_iops: ..., write_iops: ..., read_bw: ..., write_bw: ...)
         'xlarge': ///
    },
    'i3en': {
         ...
```

Of course, not suggesting to do it now.

File: dist/common/scripts/scylla_io_setup (hunk 1 above)
Comment (syuu1228) [reply dist/common/scripts/scylla_io_setup:245]: Added xlarge instance size.

**Review by yaronkaikov — APPROVED** [review]

**Comment by slivne** [comment]
@avikivity ping ...

File: dist/common/scripts/scylla_io_setup (hunk 1 above)
Comment (avikivity) [reply dist/common/scripts/scylla_io_setup:245]: I don't see xlarge.

Comment (yaronkaikov) [reply dist/common/scripts/scylla_io_setup:245]: @syuu1228 Looks like you added the `xlarge` part to the wrong commit

**Review by yaronkaikov — CHANGES_REQUESTED** [review]

**Comment by syuu1228** [comment]
@yaronkaikov Oh sorry, fixed that.

**Comment by yaronkaikov** [comment]
@avikivity `xlarge` part is in the right commit now .

**Pull request merged**